/*
 * database.h
 *
 *  Created on: Jun 10, 2020
 *      Author: jochenalt
 */

#ifndef INCLUDE_DATABASE_H_
#define INCLUDE_DATABASE_H_

#include <geometry_msgs/Pose.h>
#include <minibot/JointStateConfiguration.h>
#include <minibot/MinibotState.h>

namespace Minibot {
namespace Database {

}
}


#endif /* INCLUDE_DATABASE_H_ */
